.azure/lint_docs_qpy-linux.yml

```diff
@@ -20,14 +21,13 @@ jobs:
         displayName: 'Use Python ${{ parameters.pythonVersion }}'
 
       - bash: tools/install_ubuntu_docs_dependencies.sh
-        displayName: 'Install dependencies'
+        displayName: 'Install docs dependencies'
 
-      - bash: |
-          set -e
-          tox -e docs
-          # Clean up Sphinx detritus.
-          rm -rf docs/_build/html/{.doctrees,.buildinfo}
-        displayName: 'Run Docs build'
+      - bash: tox run -e docs,lint
```

Comment:
Oh good idea to leverage tox for lint too. Save a lot of config logic here.

Reply:
laziness is often a good motivator for single-source-of-truth configuration lol